config/dev/docker-compose.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+[[ -f .env-dev ]] && source .env-dev
+if [ -z "${MY_DOCKER_HOST}" ]; then
+  MY_DOCKER_HOST=172.17.0.1
+fi
+sed "s/\${MY_DOCKER_HOST}/$MY_DOCKER_HOST/g" ./ngnix.conf.template > ./.ngnix.conf
```

Comment:
Is het niet handiger om 'envsubst' te gebruiken?

Reply:
envsubst bestaat niet op Mac

Reply:
envsubst kan je wel installeren via brew met de package gettext.
`brew install gettext
brew link --force gettext`

Reply:
Maar sed werkt op alles, dus dat is goed 👍